src/stores/types/orderLineDetails.ts

```diff
@@ -0,0 +1,72 @@
+import { IOrderLineDetailsType } from 'models/orderLineDetails/IOrderLinedetailsType';
+import OrderLineDetailsActionTypes from 'stores/actions/orderLineDetails/orderLineDetailsTypes';
+
+export interface OrderLineDetailsState {
+  fetchingOrderLineDetails: boolean;
+  fetchedOrderLineDetails: boolean;
+  orderLineDetails: IOrderLineDetailsType [] | [];
+  orderLineDetail: IOrderLineDetailsType | null;
+  error: string | null;
+}
+
+export interface GetOrderLineDetailsRequestPayload {
+  payload: any;
+}
+
+export interface GetOrderLineDetailsSuccessPayload {
+  orderLineDetails: [IOrderLineDetailsType];
+}
+
+export interface GetOrderLineDetailsErrorPayload {
+  error: string;
+}
+
+export interface PostOrderLineDetailsRequestPayload {
+  payload: { week_days: any, id: number };
```

Comment:
I think we don't need here 'week_days'. It is a part of 'CustomerWeekDays' I assume,  not 'OrderLineDetails' Type